activerecord/test/cases/associations/belongs_to_associations_statement_cache_test.rb

```diff
@@ -0,0 +1,81 @@
+require 'cases/helper'
+
+class Dity < ActiveRecord::Base
```

Comment:
Could you try to use the current models in the Active Record test suite to reproduce this problem?